Library/Dungeon Fantasy/Dungeon Fantasy Adventure 1/Rivals/Mage - Wizard-Scholar.gcs

```diff
@@ -1245,8 +1244,9 @@
 					],
 					"calc": {
 						"level": 12,
+						"damage": "1d-3 cr",
 						"parry": "10",
-						"damage": "1d-3 cr"
+						"block": "10"
 					}
```

Comment:
Punch should not have a block score

Reply:
I think this sheet was created with an older version of the Natural Attacks. I've deleted it and re-added.